ilastik/workflows/tracking/conservation/conservationTrackingWorkflow.py

```diff
@@ -353,27 +355,56 @@ def prepare_lane_for_export(self, lane_index):
             withBatchProcessing = True
         )
 
-    def post_process_lane_export(self, lane_index):
+    def post_process_lane_export(self, lane_index, time=1):
```

Comment:
We could maybe change the parameter name to `overwriteExistingFiles` and set it to `True` by default?

@JaimeIvanCervantes I agree that changing the signature of `post_process_lane_export` is not the most elegant way, but as a default parameter is specified that shouldn't be a problem, right?

@stuarteberg: Is the export of other files from the export applets simply overwriting existing results, or is there some other way the user could be notified about files that are about to be overwritten?